Makefile

```diff
@@ -183,6 +185,11 @@ help:
 	@echo '    For tags above, the final image tag will include the registry defined by "REGISTRY".'
 	@echo '    e.g., REGISTRY="myreg" DAEMON_TAG="mydaemontag" will tag the daemon "myreg/mydaemontag"'
 	@echo ''
+	@echo '  BASE_IMAGE - Do not compute the base image to be used as container base from BASEOS_REPO'
+	@echo '               and BASEOS_TAG. Instead, use the base image specified. The BASEOS_ vars will'
+	@echo '               still be used to determine the ceph-container source files to use.'
+	@echo '               e.g., BASEOS_TAG="alpine:3.6" to use Alpine v3.6 as a base image
```

Comment:
That last line was confusing for me as it shows a normal use case while you explain the BASE_IMAGE override. Maybe adding BASE_IMAGE="myrepo/mycustomalpine:mytag" could make this example easier to understand.